src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -28,8 +33,16 @@ export async function invokeGoLambda(ctx: ExtContext, config: GoDebugConfigurati
     // eslint-disable-next-line @typescript-eslint/unbound-method
     config.onWillAttachDebugger = waitForDelve
 
-    if (!config.noDebug) {
-        await installDebugger(config.debuggerPath!)
+    if (!config.noDebug && !(await installDebugger(config.debuggerPath!))) {
+        showErrorWithLogs(
+            localize(
+                'AWS.sam.debugger.godelve.failed',
+                'Failed to install Delve for the lambda container. See logs for details.'
```

Comment:
since there is a "View Logs" button we don't need it in the message.
```suggestion
                'Failed to install Delve for the lambda container.'
```